pkg/connectors/datacatalog/clients/datacatalog_grpc.go

```diff
@@ -106,35 +107,37 @@ func ConvertDataCatalogGrpcRespToOpenAPIResp(result *pb.CatalogDatasetInfo) (*da
 
 	//nolint:revive // Ignore add-constant to log msg
 	for colName, compMetaData := range result.GetDetails().Metadata.GetComponentsMetadata() {
-		if compMetaData != nil {
-			rscCol := datacatalog.ResourceColumn{
-				Name: colName}
-			rsColMap := make(map[string]interface{})
-			tags := compMetaData.GetTags()
-			for i := 0; i < len(tags); i++ {
-				rsColMap[tags[i]] = true
-			}
-
-			responseBytes, errJSON := json.MarshalIndent(rsColMap, "", "\t")
-			if errJSON != nil {
-				return nil, fmt.Errorf("error Marshalling in ConvertDataCatalogGrpcRespToOpenAPIResp: %v", errJSON)
-			}
-			log.Print("responseBytes after MarshalIndent in ConvertDataCatalogGrpcRespToOpenAPIResp:" + string(responseBytes))
-
-			if err := json.Unmarshal(responseBytes, &rscCol.Tags); err != nil {
-				return nil, fmt.Errorf("error UnMarshalling in ConvertDataCatalogGrpcRespToOpenAPIResp: %v", errJSON)
-			}
-
-			// just printing - start
-			responseBytes, errJSON = json.MarshalIndent(&rscCol, "", "\t")
-			if errJSON != nil {
-				return nil, fmt.Errorf("error Marshalling in ConvertDataCatalogGrpcRespToOpenAPIResp: %v", errJSON)
-			}
-			log.Print("responseBytes after MarshalIndent in ConvertDataCatalogGrpcRespToOpenAPIResp:" + string(responseBytes))
-			// just printing - end
-
-			resourceCols = append(resourceCols, rscCol)
+		if compMetaData == nil {
```

Comment:
how do all these changes relate to the PR?

Reply:
This was fixed as `golangci-lint` was complaining about this error.